sdks/python/src/opik/cli/export_dataset.py

```diff
@@ -0,0 +1,238 @@
+"""Dataset export functionality."""
+
+import sys
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+import click
+from rich.console import Console
+
+import opik
+from opik.api_objects.dataset.dataset import Dataset
+from opik.cli.export_utils import (
+    debug_print,
+    dataset_to_csv_rows,
+    should_skip_file,
+    write_csv_data,
+    write_json_data,
+    print_export_summary,
+)
+
+console = Console()
+
+
+def export_single_dataset(
+    dataset: Dataset,
+    output_dir: Path,
+    max_results: Optional[int],
+    force: bool,
+    debug: bool,
+    format: str,
+) -> int:
+    """Export a single dataset."""
+    try:
+        # Check if already exists and force is not set
+        if format.lower() == "csv":
+            dataset_file = output_dir / f"datasets_{dataset.name}.csv"
```

Comment:
shouldn't "datasets_" be "dataset_" in csv case? Or multiple datasets can be merged somehow in the same csv?